test/validation/api/classification/odp_classification_common.c

```diff
@@ -107,8 +107,23 @@ int cls_pkt_set_seq(odp_packet_t pkt)
 	CU_ASSERT_FATAL(offset != ODP_PACKET_OFFSET_INVALID);
 
 	if (ip->proto == ODPH_IPPROTO_UDP)
-		status = odp_packet_copy_from_mem(pkt, offset + ODPH_UDPHDR_LEN,
-						  sizeof(data), &data);
+		if (odp_packet_has_vxlan(pkt)) {
+			/* TODO: Inner header parsing is not supported
+			 * in this ODP release. Hence the packet parsing
+			 * is skipped. This code has to be modifies once
```

Comment:
Typo: modified

Reply:
The "TODO" can't stand by itself. Sounds like we need an additional `odp_classification_capability()` bit to indicate whether inner packets are included in the parse?

Reply:
Even if we add the capability currently we don't have ODP APIs expose them as packet metadata functions. i.e odp_packet_has_inner_tcp() 

Should we add the capability and defer this to after tiger-moth?